src/Microsoft.OData.Client/BaseSaveResult.cs

```diff
@@ -262,7 +262,19 @@ internal static DataServiceClientException GetResponseText(Func<Stream> getRespo
                     headers.Add(ODataConstants.ContentTypeHeader, JsonContentTypeHeader);
                     httpWebResponseMessage = new HttpWebResponseMessage(headers, (int)statusCode, getResponseStream);
                     ODataMessageReader messageReader = new ODataMessageReader(httpWebResponseMessage);

```

Comment:
Do we need to dispose the MessageReader?

Reply:
IIRC message reader would be automatically disposed once out of scope - I'm guessing it was because it was created from the underlying `getResponseStream`